.github/workflows/restart.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: restart
         uses: usds/cloud-gov-cli@master
         with:
-          command: restart dashboard --strategy rolling
+          command: restart dashboard-stage --strategy rolling
```

Comment:
Our other apps follow the convention of having the same name for all environments (`dashboard` in this case). Why are we doing something different for dashboard?

Reply:
The newrelic extension in the PHP bundle doesn't allow overriding the app name.  I paired with @jbrown-xentity on this approach for a unique app name to use with newrelic.  This [PR](https://github.com/GSA/project-open-data-dashboard/pull/326/files) has the related changes.

These are the newrelic defaults that can be overridden.
https://github.com/cloudfoundry/php-buildpack/blob/master/extensions/newrelic/extension.py#L27-L33